client/homebrew/editor/snippetbar/snippetbar.less

```diff
@@ -46,6 +46,30 @@
 					color : black;
 				}
 			}
+			&.toTop{
```

Comment:
I guess we don't need these styles anymore with the snippetbar buttons removed.

Reply:
Agreed; pruning them now.